Adding CSV panel actions for SavedSearch embedabbles

joelgriffith opened the pull request:
This includes Tim's work as well, but wanted to get it out for early review as I had to alter a few utilities to make this work (I'll notate them).

Commit:
Adding CSV panel actions for SavedSearch embedabbles

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/3763/)

tsullivan reviewed (changes requested):
I commented on a few things that might need attention before merge

joelgriffith commented on x-pack/plugins/reporting/public/panel_actions/get_csv_panel_action.tsx:
```diff
@@ -0,0 +1,116 @@
+      timerange: {
+        min: fromTime.valueOf(),
+        max: toTime.valueOf(),
+        timezone: 'PST',
```
Crud, I definitely had a fix for this...

tsullivan replied:
`config.get('dateFormat:tz')`

Commit:
WIP: Implementing PR feedback

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/5679/)

Commit:
Fixing imports, timzeone config, and more for exporting CSV

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/5916/)

joelgriffith commented:
retest

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6017/)

joelgriffith commented:
retest

joelgriffith commented:
Just merged master into this base branch and pushed, retesting to see if build now works.

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6049/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6060/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6079/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6216/)

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6228/)

alisonelizabeth committed:
refetch autocomplete info after updating dev console settings (#32587)

* refetch autocomplete info after updating dev console settings

* address review feedback

* only update autocomplete settings if user has changed settings

* refactor code to only refetch autocomplete settings on change

* simplify logic based on review feedback

* address review feedback

alvarezmelissa87 committed:
Add missing whitespace to modelPlotCardinality + cardinalityNotValid messages (#33599)

gchaps committed:
[DOCS] Adds information about enabling/disabling rollover (#33600)

Commit:
Duck typing vs instanceof checks

joelgriffith commented:
This is somewhat blocked on https://github.com/elastic/eui/pull/1737

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6258/)

joelgriffith commented:
```../../../../../src/legacy/ui/public/saved_objects/components/saved_object_finder.tsx:37:3 - error TS2305: Module '"@elastic/eui"' has no exported member 'EuiListGroupItem'.```

Is the failure, which is accurate

cqliu1 committed:
[Canvas] Show current workpad in workpad loader (#33531)

* Bolds loaded workpad in workpad loader

* Extracted helper function

monfera committed:
Refactor: make interaction target acquisition faster and more robust (#33503)

ppisljar committed:
adding runPipeline tests (#27015)

lizozom committed:
New platform agg_types deps (#33372)

* Moved auto_select_if_only_one directive

* Moved validate-date-interval directive

* Moved scroll-bottom directive

* Moved input number directive + Fix validate-date-interval directive

* Moved validate ip and validate cidr mask directives

* Moved validate date math and documentation href directives

* Moved validate agg directive

* Moved click focus directive

* removed validate date interval from autoload

stacey-gammon commented:
I'm hitting this too in https://github.com/elastic/kibana/pull/33602.  But master is green.  Maybe we need to rebase.

alexwizp committed:
[TSVB] Table View - critical issue after cleaning the 'Group by' field (#33645)

lizozom committed:
New platform dropdown (#33520)

* Removed old dropdown code from dev tools consle

* Removed old dropdown from vega tools

* Deleted dropdown directives from angular bootstrap

* Deleted ui.bootstrap.dropdown test (irrelevant, since directive was deleted)

gchaps committed:
[DOCS] Removes beta tag from Canvas docs (#33630)

azasypkin committed:
Typesciptify SAML Authentication provider. (#33559)

joelgriffith commented:
@stacey-gammon I think the issue is that master isn't hitting that codepath, as I have tried numerous times to rebase master (they're on the same version). I'm going to see if we can push through this PR.

monfera committed:
[Canvas] Fix: no stuck hover on key based page back/forth (#33631)

* Fix: no stuck hover on key based page back/forth

* Fix: no stuck hover on key based page back/forth

joshdover committed:
Replace `start` with `setup` lifecycle event (#33455)

* Move `start` lifecycle functions to `setup` in core/server

* Move `start` lifecycle functions to `setup` in core/public

peterschretlen committed:
Fix clipping of wide vega tooltip tables (#33465)

* fix wide vega table tooltips, desktop and mobile (#33436)

ogupte committed:
[APM] Related error count in transaction timeline (#33100)

* Closes #29688 by adding related error count in timeline with links to error
overview filtered by the related trace/transaction ids. Also update the errors
link in the sticky transaction properties to match the timeline pattern.

* [APM] leverage EUI theme colors, tweak sticky transaction properties column
sizes, and reorder timeline error count to show duration after only on hover

* [APM] Renamed ErrorsOverviewLink -> ErrorCountBadge. Differentiate the labeled
vs non-labeled variaions of ErrorCountBadge. Replace state hooks for hovered
styles with plain css :hover

* [APM] Revert duration display on hover to render display all the time.

* remove unnecessary props object spread

weltenwort committed:
[Infra + Logs UI] Add source configuration ui to the docs (#33535)

This adds the new source configuration ui to the documentation of both the Infrastructure UI and the Logs UI. It also removes the `BETA` badge to reflect the status change.

fixes elastic/kibana#31124

Co-authored-by: Brandon Morelli <bmorelli25@gmail.com>

jasonrhodes committed:
Upgrades EUI to 9.5 (#33672)

* Updates EUI to 9.5.0

* Remove duplicate type

* Updates snapshots and fixes a few type errors

* Updates x-pack snapshots

sorenlouv committed:
[APM] Fix Datepicker double loading and move date parsing to urlParams (#33560)

Commit:
Merge remote-tracking branch 'upstream/master' into feature/reporting/csv-export-panel-action

joelgriffith commented:
I'm not sure why this PR isn't showing the right deltas, but I've re-opened a new PR here: https://github.com/elastic/kibana/pull/33709

elasticmachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/6474/)
